Packs/CortexXDR/Layouts/layoutscontainer-Cortex_XDR_Incident.json

```diff
@@ -842,6 +884,344 @@
+                                "startCol": 0
+                            },
+                            {
+                                "endCol": 2,
+                                "fieldId": "operationtype",
+                                "height": 22,
+                                "id": "f8abcf60-158f-11ee-9c72-eb2e3cb3cb42",
+                                "index": 2,
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            },
+                            {
+                                "endCol": 2,
+                                "fieldId": "identitytype",
+                                "height": 22,
+                                "id": "fc49ddb0-158f-11ee-9c72-eb2e3cb3cb42",
+                                "index": 3,
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            },
+                            {
+                                "endCol": 2,
+                                "fieldId": "region",
+                                "height": 22,
+                                "id": "6204ffb0-15b6-11ee-9c72-eb2e3cb3cb42",
+                                "index": 4,
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            },
+                            {
+                                "endCol": 2,
+                                "fieldId": "resourcetype",
+                                "height": 22,
+                                "id": "69d67e80-15b6-11ee-9c72-eb2e3cb3cb42",
+                                "index": 5,
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            },
+                            {
+                                "endCol": 2,
+                                "fieldId": "referencedresourceid",
+                                "height": 22,
+                                "id": "a6834e20-18be-11ee-a005-5968c8362785",
+                                "index": 6,
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            },
+                            {
+                                "endCol": 2,
+                                "fieldId": "referencedresourcename",
+                                "height": 22,
+                                "id": "a7935e90-18be-11ee-a005-5968c8362785",
+                                "index": 7,
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            }
+                        ],
+                        "maxW": 3,
+                        "minH": 1,
+                        "moved": false,
+                        "name": "Cloud Extra Data",
```

Comment:
Maybe remove the extra? why its extra data? (i know the relevant command is get cloud extra data)

Reply:
Cloud Info?